src/frontend/src/components/AddUser/index.tsx

```diff
@@ -0,0 +1,248 @@
+          <h1 className="mb4 f4 fw6">Add User</h1>
+          <form onSubmit={this.handleSubmit} noValidate={true}>
+            <legend className="visually-hidden">Sign Up</legend>
+            <div className="mb4">
+              <label htmlFor="username" className="db mb2 fw6">
+                Name
+              </label>
+              <input
+                id="username"
+                name="username"
+                type="text"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.invalidCredentials
+                    ? 'no_match_msg'
+                    : this.state.missingPassword
+                    ? 'input_msg'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.invalidCredentials || this.state.missingPassword
+                    ? true
+                    : false
+                }
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb4">
+              <label htmlFor="email" className="db mb2 fw6">
+                Email
+              </label>
+              <input
+                id="email"
+                name="email"
+                type="email"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.invalidEmail
+                    ? 'email_msg'
+                    : this.state.invalidCredentials
+                    ? 'no_match_msg'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.invalidEmail || this.state.invalidCredentials
+                    ? true
+                    : false
+                }
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb4">
+              <label htmlFor="password" className="db mb2 fw6">
+                Password
+              </label>
+              <input
+                id="password"
+                name="password"
+                type="password"
+                className="w-100 pa3 br2 b--black-20"
+                required={true}
+                aria-describedby={
+                  this.state.invalidCredentials
+                    ? 'no_match_msg'
+                    : this.state.missingPassword
+                    ? 'input_msg'
+                    : undefined
+                }
+                aria-invalid={
+                  this.state.invalidCredentials || this.state.missingPassword
+                    ? true
+                    : false
+                }
+                onChange={this.handleChange}
+              />
+            </div>
+            <div className="mb4">
+              <label htmlFor="group" className="db mb2 fw6">
+                Group
+              </label>
+              <input
```

Comment:
In the prototype this field looks more like the way you have created it in the `EditUser` component. I am not sure which one makes more sense for the end goal of the application, this may be a good thing to talk about in more detail at the next meetup. 

I would say just make them both the same for now, either `text` or `select`.

Reply:
I don't think we need to require group for now, but yeah Michael should weigh in. It should be a `select` offering the current groups.

Reply:
Michael isn't concerned with groups as a feature for now. So we can just leave it as is, as a field that exists but is optional.

Reply:
And @hmarcks is right: when we make groups an actual feature it needs to have its own defined entity rather than a free string input field, including the backend support: #364